packages/vue-cli-plugin-typescript/generator/template/README.md

```diff
@@ -1,8 +1,8 @@
-# <%- rootOptions.projectName %>
+# @coveo/vue-cli-plugin-typescript
```

Comment:
You should keep the `rootOptions.projectName` it will be substituted by the end user's project name when the end-user uses the template.

Reply:
ah that makes sense, thanks!